qir/qsharp-target-tests/template/Example.csproj

```diff
@@ -1,19 +1,10 @@
-<Project Sdk="Microsoft.Quantum.Sdk/0.24.41925-alpha">
-
+<Project Sdk="Microsoft.Quantum.Sdk/{{version}}{{channel}}">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
-    <QirGeneration>true</QirGeneration>
-    <AdditionalQscArguments>--runtime-override AdaptiveExecution</AdditionalQscArguments>
-    <IncludeCSharpRuntime>False</IncludeCSharpRuntime>
-    <IncludeQSharpCorePackages>False</IncludeQSharpCorePackages>
     <IncludeProviderPackages>False</IncludeProviderPackages>
-    <CSharpGeneration>False</CSharpGeneration>
-
+    <ExecutionTarget>{{executionTarget}}</ExecutionTarget>
+    <QirGeneration>true</QirGeneration>
```

Comment:
This isn't needed unless you want the human readable .ll output. The bitcode is produced without it and can be found in the obj/qsharp folder of the project. 
```suggestion
```

Reply:
It is needed for test output